pkgs/development/tools/hotdoc/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, python3Packages
+, pkg-config
+, cmake
+, glib
+, json-glib
+, libxml2
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "hotdoc";
+  version = "0.12.2";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0nypvcqp4b0qfps35f7mz3d90zjh0v5wawpd06ra28q92nxblx17";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  preConfigure =
+    # Cmake is needed to build a c extension CMARK, that's found inside the
+    # source tarball. We add cmake to PATH here, and not in nativeBuildInputs,
+    # since it's not as the main build system of the project.
+  ''
+    export PATH="$PATH:${lib.getBin cmake}/bin"
```

Comment:
That does not change the fact that `cmake` is a build dependency. You will just need to use `dontUseCmakeConfigure = true;`.

Reply:
Good to know.